_includes/header.html

```diff
@@ -33,7 +33,13 @@
           </form>
           <ul class="nav navbar-nav navbar-right">
             <li{% if page.nav == "docs" %} class="active"{% endif %}>
-              <a href="{{ site.docs_site_url }}">Documentation</a>
+              <a href="/versions/{{ current_version}}{{ site.getting_started_category_url }}">Getting Started</a>
```

Comment:
Add a space in  `{{ current_version}}`
And maybe on category_url
And again at line 42